tests/www/apply/test_submit.py

```diff
@@ -4959,7 +4959,12 @@ def test_with_previous_as_job_seeker(self, client):
             response, "Vous avez déjà postulé chez cet employeur durant les dernières 24 heures.", status_code=403
         )
 
+        # Don't allow to skip to another step
+        response = client.get(self.application_jobs_url)
+        assert response.status_code == 404
+
         # Make it less recent to avoid the 403
+        self._login_and_setup_session(client, self.job_seeker)
```

Comment:
Je mettrais plutôt le `self._login_and_setup_session(client, self.job_seeker)` après le `job_application.save(...)` (2 lignes plus bas) pour laisser le commentaire `# Make it less recent to avoid the 403` à coté de la modif.
(et pareil pour les 2 autres tests).